src/main/java/org/opensearch/knn/index/query/KNNWeight.java

```diff
@@ -285,12 +286,13 @@ private Map<Integer, Float> doANNSearch(final LeafReaderContext context, final i
     }
 
     private Map<Integer, Float> doExactSearch(final LeafReaderContext leafReaderContext, final int[] filterIdsArray) {
+        log.error("Doing Exact search");
```

Comment:
is this right that logging is at ```error``` level?

Reply:
This is interesting. I removed it from my local, but it is still coming. Let me double check.